src/commands/farm.rs

```diff
@@ -67,7 +67,7 @@ pub(crate) async fn farm(is_verbose: bool, executor: bool) -> Result<()> {
         }
     }
 
-    let summary = Summary::new(Some(farmer_config.plot_size)).await?;
+    let summary = Summary::new(Some(farmer_config.clone().plot_size)).await?;
```

Comment:
Do you need clone here though :thinking: ?

Reply:
yes, `copy` is not implemented for `bytesize` wrapper on sdk. No big deal, we can fix it later :)

Reply:
Okay, can you move clone to `plot_size` then? Like `farmer_config.plot_size.clone()`?

Reply:
lemme try sure